tests/functional/arxiv/fixtures/arxiv_smoke_record.json

```diff
@@ -56,6 +56,11 @@
         "value": "Proc. of CRIS2015, Gallipoli, Italy, 8 pages, 17 figures, to appear\n  in Elsevier in Nuclear and Particle Physics Proceedings (2016)"
       }
     ],
+    "publication_info": [
+      {
+        "material": "publication"
+      }
+    ],
```

Comment:
We should not populate the `publication_info` if we only have `material`. We should have though an example of arxiv record with some publication info.